### ISSUE TITLE
Add term 3-5 finger cutaneous syndactyly

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10329]
**Preferred term label:**
3-5 finger cutaneous syndactyly

**Synonyms**
Soft tissue syndactyly of fingers 3-5, Webbed 3th, 4th and 5th finger - Webbed third, fourth and fifth finger

**Definition (free text, please give PubMed ID)**
A soft tissue continuity in the anteroposterior axis between the third to the fifth fingers that extends distally to at least the level of the proximal interphalangeal joints.
PMID: 37352860

**Parent term (use hpo.jax.org/app)**
Cutaneous finger syndactyly HP:0010554

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
For example: PMID: 37352860

**Your nano-attribution (ORCID)**
A. Zankl 0000-0001-8612-1062, @azankl
L. Wit 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
Adding new term: 
- ID: HP:6000505
- label: 3-5 finger cutaneous syndactyly
- parents: [Cutaneous finger syndactyly (HP:0010554)](https://hpo.jax.org/app/browse/term/HP:0010554) | [3-5 finger syndactyly (HP:0010710)](https://hpo.jax.org/app/browse/term/HP:0010710)
- pmid: PMID:37352860
- def: A soft tissue continuity in the anteroposterior axis between the third to the fifth fingers that extends distally to at least the level of the proximal interphalangeal joints.
- comment: 
- orcid: orcid:0009-0007-1020-5412
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10329
- synonyms: Webbed 3th, 4th, and 5th finger(NONE)|Soft tissue syndactyly of fingers 3-5,(NONE)|Webbed third, fourth, and fifth finger(NONE)
